SConstruct

```diff
@@ -20,15 +20,19 @@ sources = []
 if env["platform"] == "windows":
     env.Append(CPPPATH=["thirdparty/opus/include"], LIBS=["thirdparty/opus/build/Release/opus.lib"])
     env.Append(LINKFLAGS = ['/WX:NO'])
-elif env["platform"] == "linux" or env["platform"] == "javascript":
+else:
     env.Append(CPPPATH=["thirdparty/opus/include"], LIBS=["opus"], LIBPATH="thirdparty/opus/build")
     env.Append(CXXFLAGS = ['-fpermissive'])
```

Comment:
I was able to successfully build and run without `-fpermissive` on mac m1 sonoma 14.3
```suggestion
    if env["platform"] != "macos":
        env.Append(CXXFLAGS = ['-fpermissive'])
```